cypress/integration/tests/athena_reports_testing.spec.js

```diff
@@ -0,0 +1,29 @@
+import * as AthenaReportsHelper from '../../support/helpers/athenaReportHelper';
+import learnersReportPageElements from "../../support/elements/learners_report_page_elements";
+import researcherReportsPageElements from '../../support/elements/researcher_reports_page_elements';
+import * as C from "../../support/constants";
+
+
+const RESEARCHER_REPORT_URL = 'https://researcher-reports.concord.org/branch/master/?portal=https%3A%2F%2Flearn-report.staging.concord.org';
+
+context("Athena Reports Testing", () => {
+
+    it('Verify Learners Report with 1 school, 1 teacher and 1 runnable', () => {
+
+        const REPORT_URL = 'https://learn-report.staging.concord.org/api/v1/report_learners_es/external_report_query_jwt?schools=99117&teachers=362&runnables=externalactivity_1606';
```

Comment:
Maybe you want to extract the schools, teachers, and runnables parameter value into variables or constants. If you want to run this test on a different portal those will be change.

Maybe you also want to use the UI to select the actual school, teacher, and runnable and then assert that the counts at the top of the page match what you expect.

It is useful to know if those counts match. This approach might also provide a way to determine the ids, without hardcoding them here.

Reply:
Good point. Will make those changes, Scott.

Reply:
created a test data file for Athena tests to configure these name and ids and refactored code to use that.

Reply:
Also validating the UI counters now.